app/views/user_recipes/index.html.erb

```diff
@@ -2,51 +2,39 @@
 <%= render 'shared/scroll-days' %>
 
 
-<!-- <h1><%= @day.strftime('%A %e %B') %></h1>
-<h2>Recipe of the day</h2>
-<%= simple_form_for [current_user, @user_recipe] do |f|%>
-  <%= f.input :date,  :as => :hidden, :input_html => {:value => @day} %>
-  <%= f.input :number_of_people %>
-  <%= f.association :recipe %>
-  <%= f.submit "Define recipe of the day", class: "btn btn-primary" %>
-<% end %>
-<h6 class="text-center"><%= @day.strftime('%A %e %B') %></h6>
-<div class="container">
-  <div class="row">
-    <% @user_recipes.each do |u_recipe| %>
-      <div class="col-xs-12 col-sm-6 col-md-4">
-        <% recipe = Recipe.find(u_recipe.recipe_id) %>
-        <h3><%= recipe.name %></h3>
-        <p>for <%= u_recipe.number_of_people %></p>
-        <p><%= recipe.recipe_type %></p>
-        <p><%= recipe.instructions %></p>
-      </div>
-    <% end %>
-  </div>
-</div> -->
-
+<h1 id="calender-date"><%= @day.strftime('%a %e %b') %></h1>
 <div class="meal-type">
```

Comment:
@lukecharny  we need to review the indentation here... there are many div that doesn't close where they should close